cypress.config.js

```diff
@@ -2,6 +2,7 @@ const { defineConfig } = require('cypress');
 
 module.exports = defineConfig({
   e2e: {
+    baseUrl: 'https://demoqa.com/',
```

Comment:
You need to remove "/" after .com. It may create a link like https://demoqa.com//login with two "/". Because you use cy.visit('/login'); with "/" at the beginning. It isn't a big deal but better to remove it

Reply:
Ok, I will fix it. thank you.